tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs

```diff
@@ -8,6 +8,9 @@ const abstractLang = "en";
 const USELESS_IN_INFO_PANEL = "see info panel";
 
 function addDeferredHelpTo(fragment, helpHtml, resource, translationHint) {
+  if (helpHtml) {
+    helpHtml = linkifyHttpsUrls(helpHtml);
```

Comment:
fine for now, pending a better solution in https://unicode-org.atlassian.net/browse/CLDR-15708

Reply:
I agree about auto-linkification not being the best way, still I think it's better than expecting ST users to copy and paste links. Perfect is the enemy of good.

Reply:
Yes, and I'm glad to approve this and improve things for users.